src/meta/model/migration/src/m20241125_043732_connection_params.rs

```diff
@@ -10,7 +12,11 @@ impl MigrationTrait for Migration {
             .alter_table(
                 Table::alter()
                     .table(Connection::Table)
-                    .add_column(ColumnDef::new(Connection::Params).binary().not_null())
+                    .add_column(
+                        ColumnDef::new(Connection::Params)
+                            .rw_binary(manager)
+                            .not_null(),
+                    )
```

Comment:
Is it correct to directly change an existing migration script? Not sure, just want to confirm.

Reply:
Only for MySQL, I think it's okay.